### DISCUSSION TITLE
how to create a service?

### DISCUSSION THREAD

**Discussion by split-brain-83** [ep1cman/unifi-protect-backup#55] [Q&A]
Hi and big up for creating this cool tool!
I'm running it on Synology NAS and it's woring fine. 
I'm now wondering, how to make it a service or a daemon?
There is no systemd running on my system tho, but I want the process to start up after boot, keep it running all the time and restart it if necessary.
Would you have some recommendation on how to make a such service?

**Comment by ep1cman** [comment]
Sorry, I have never used a Synology device so cant provide any specific advice in this case.

**Reply by split-brain-83** [reply]
I was able to handle it using Task Scheduler in DSM and its running fine.
